### DISCUSSION TITLE
Dynamically bind a specific cloudlet with a vm

### DISCUSSION THREAD

**Discussion by Ljincheng0128** [manoelcampos/cloudsim-plus#370] [Q&A] [answered]
hello，
Can I dynamically submit cloudlets at different time when a task arrives? I would like to dynamically bind a specific cloudlet with a vm, considering the current state of the vm list (for example, whether there are running cloudlets or cloudlets waiting to be executed on current vm). Thanks.

**Comment by manoelcampos** [comment, marked as answer]
That is totally possible. You just have to define your programming logic. Check the DynamicCloudletsArrival examples available [inside the examples project](https://github.com/manoelcampos/cloudsimplus/tree/master/cloudsim-plus-examples/src/main/java/org/cloudsimplus/examples/dynamic).
